00-RELEASENOTES

```diff
@@ -1,16 +1,155 @@
+* Support RDMA for valkey-cli and benchmark by @pizhenwei (#2059)
+* Add multi-database support to cluster mode by @xbasel (#1671)
+* Introduce MPTCP for replica by @pizhenwei (#1961)
+* Add --hotkeys-count option for Valkey-cli by @hwware (#1933)
+* Add sentinel_total_tilt to sentinel INFO sentinel by @carlosfu (#1904)
+* Introduce MPTCP by @pizhenwei (#1811)
 
-    https://valkey.io/download/
+### New Configs
+* Auto-failover on shutdown unified config by @zuiderkwast (#2292)
 
-More information is available at https://valkey.io
+### New info fields and Logging
+* [NEW] Introduce lttng based tracing by @artikell (#2070)
+* Add node pfail and fail count to cluster info metrics by @hpatro (#1910)
 
-Happy hacking!
+### Performance/Efficiency Improvements
+* Optimize scan/sscan/hscan/zscan commands by replacing list with vector by @chzhoo (#2160)
+* Allow shrinking hashtables in low memory situations by @Fusl (#2095)
+* Reply Copy Avoidance by @xbasel (#2078)
+* Combine range element ranks calculation with range elements search to improve zcount performance by @SoftlyRaining (#2129)
+* Replication Flow Control – Prioritizing replication traffic in the replica by @xbasel (#1838)
+* Optimize GEORADIUS command performance with  pre-allocated buffer by @chzhoo (#2116)
+* Optimize string2ll with load-time CPU feature check using IFUNC resolver by @zhulipeng (#2099)
+* Optimize WATCH by equalStringObjects early length check by @vitahlin (#2107)
+* Optimize BITCOUNT using ARM NEON SIMD by @xbasel (#1867)
+* Optimize findBucket performance using SIMD by @zhulipeng (#2030)
+* Hyperloglog ARM NEON SIMD optimization by @xbasel (#1859)
+* Accept limited number of new cluster link connections per cycle by @hpatro (#2009)
+* Optimize string-to-integer performance using AVX512 by @zhulipeng (#1944)
+* Optimize bitcount command by SIMD by @chzhoo (#1741)
+* Embed hash value in hash type entry by @zuiderkwast (#1579)
+
+### Cluster
+* Add CLUSTER FLUSHSLOT command by @wuranxx (#1384)
+* Trigger manual failover on SIGTERM / shutdown to cluster primary by @enjoy-binbin (#1091)
+* Implement: CLUSTER REPLICATE NO ONE by @skolosov-snap (#1674)
+* Add cluster-manual-failover-timeout to configure the timeout for manual failover by @enjoy-binbin (#1690)
+* Add Cluster Bus Port out of range error message for Cluster Meet command by @hwware (#1686)
+
+### Module
+* Add new module API flag to bypass command validation by @sungming2 (#1357)
+
+### Bug Fixes
+* Redact user data when hide-user-data-from-log enabled by @YueTang-Vanessa (#2274)
+* Prevent bad memory access when NOTOUCH client gets unblocked by @uriyage (#2347)
+* Fix large allocations crashing Valkey during active defrag by @Fusl (#2353)
+* Fix replica (the old primary) claims to sitll have slots after manual failover by @enjoy-binbin (#2301)
+* Change default values of valkey-cli to use valkey naming, and added fallback to old values by @avifenesh (#2334)
```

Comment:
Probably goes to tooling. Not really a bug fix, it's more of rebranding redis -> valkey stuff.